extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/exec/WorkerImpl.java

```diff
@@ -520,6 +525,7 @@ public InputStream readChannel(
   @Override
   public void postWorkOrder(final WorkOrder workOrder)
   {
+    log.info("Got work order for stage[%d]", workOrder.getStageNumber());
```

Comment:
Nit: better to use standard English spacing: a space between "stage" and the bracket: `stage [%d]`.

The brackets are really only needed for items that can contain spaces, so we know what is the message and what is the value. But, this is a number. so `state %d`.

Reply:
In MSQ, in most places we are using this `[]` convention. I would hate to change it partially. Hence, if we decide to move away from using no brackets for numbers, then we should do it as part of another pr which touches all MSQ code.